{"name": "test_to_login", "status": "broken", "statusDetails": {"message": "RecursionError: maximum recursion depth exceeded", "trace": "self = <test.test_web.test_to_login.Test_HomePage object at 0x000001C3C091C9A0>\n\n    def test_to_login(self):\n        self.home_proxy.to_login()\n>       login_page_title = self.login_proxy.get_title()\n\ntest\\test_web\\test_to_login.py:24: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\npage\\login_page.py:34: in get_title\n    return self.login_handle.get_title_info()\npage\\login_page.py:26: in get_title_info\n    return self.get_text(self.get_title_info())\npage\\login_page.py:26: in get_title_info\n    return self.get_text(self.get_title_info())\nE   RecursionError: maximum recursion depth exceeded\n!!! Recursion detected (same locals & position)"}, "attachments": [{"name": "stdout", "source": "03dd395b-06e1-490a-89b7-2c3c4865d199-attachment.txt", "type": "text/plain"}], "start": 1668533667987, "stop": 1668533670039, "uuid": "9e2d677c-9f5b-4ffe-b808-f07f8efd0854", "historyId": "12ec9096dc2532553bb49276bea8f21d", "testCaseId": "fba31f286f30d766dc7244a52e88fd51", "fullName": "test.test_web.test_to_login.Test_HomePage#test_to_login", "labels": [{"name": "parentSuite", "value": "test.test_web"}, {"name": "suite", "value": "test_to_login"}, {"name": "subSuite", "value": "Test_HomePage"}, {"name": "host", "value": "xiaoma"}, {"name": "thread", "value": "27924-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test.test_web.test_to_login"}]}